src/actions/projects.js

```diff
@@ -105,6 +106,8 @@ export function fetchProjects() {
                         dispatch(addProjectsFailure);
                         reject(error)
                     })
+            }).catch(() => {
+                // Stub out avatar rejection because it is optional for users to have avatars
```

Comment:
I think this is alright. Normally I would at least add a `console.warn` or something similar here, but that's not as helpful for the mobile app.

Reply:
Yeah, in this case I think we're okay because a user having an avatar really _should_ be optional. Having no avatar doesn't affect their ability to engage with the core functions of the app (working on projects), and from a data ethics perspective any user data that we don't absolutely need should be optional.

If it _were_ something that affected some app functionality in some way, that's where I'd throw up a warning.